Missing return type of resize_connections() function

https://github.com/jakobj/nest-simulator/blob/f0c976029c5612c68b902d83c05f51045b317808/nestkernel/connection_manager.cpp#L1646

This function is missing the `void` return type in the cpp-file. Therefore I cannot compile the 5g branch at the moment:
```
/home/hahne/git/nest-simulator/nestkernel/connection_manager.cpp:1646:45: error: ISO C++ forbids declaration of ‘resize_connections’ with no type [-fpermissive]
 nest::ConnectionManager::resize_connections()
                                             ^
/home/hahne/git/nest-simulator/nestkernel/connection_manager.cpp:1646:1: error: prototype for ‘int nest::ConnectionManager::resize_connections()’ does not match any in class ‘nest::ConnectionManager’
 nest::ConnectionManager::resize_connections()
 ^~~~
In file included from /home/hahne/git/nest-simulator/nestkernel/connection_manager.cpp:23:0:
/home/hahne/git/nest-simulator/nestkernel/connection_manager.h:412:8: error: candidate is: void nest::ConnectionManager::resize_connections()
   void resize_connections();
        ^~~~~~~~~~~~~~~~~~
```

--- a/nestkernel/connection_manager.cpp
+++ b/nestkernel/connection_manager.cpp
@@ -1643,6 +1643,7 @@ nest::ConnectionManager::remove_disabled_connections( const thread tid )
   }
 }
 
+void
 nest::ConnectionManager::resize_connections()
 {
   kernel().vp_manager.assert_single_threaded();

